src/mlpack/core/data/load_impl.hpp

```diff
@@ -393,6 +406,39 @@ bool Load(const std::string& filename,
   return true;
 }
 
+// For loading data into sparse matrix
+template <typename eT>
+bool Load(const std::string& filename,
+          arma::SpMat<eT>& matrix,
+          const bool fatal,
+          const bool transpose)
+{
+  bool success;
+  Timer::Start("loading_data");
+  success = matrix.load(filename, arma::coord_ascii);
```

Comment:
@rcurtin 

I tried using arma::csv_ascii instead of arma::coord_ascii, here
Both the new tests which I wrote namely : SaveSparseCSVTest, and LoadSparseCSVTest have failed.
It seems that arma::coord_ascii is the only method to load a narrow matrix.

(ARMA version: 8.400.0 (Entropy Bandit))

Reply:
Ah, sorry, I think you are right here.  I misread the output of my experiments.  So then in this case I would say that we should use `coord_ascii` for a `.csv` file.  Later, we can figure out how to let the user specify the type manually.